CMakeLists.txt

```diff
@@ -50,10 +50,21 @@ Please set this to environment variable to point to the LLVM build directory\
 else()
         set(IN_SOURCE_BUILD 1)
 endif()
-set(Z3_DIR $ENV{Z3_DIR})
+
+if (DEFINED ENV{Z3_DIR})
+	get_filename_component(Z3_DIR $ENV{Z3_DIR} REALPATH BASE_DIR ${CMAKE_BINARY_DIR})
+    set(Z3_LIBRARIES ${Z3_DIR}/bin/libz3.a)
+	set(Z3_INSTALLS ${Z3_DIR}/include)
+	set(Z3_EXTRA_LINK_DIR ${Z3_DIR}/include)
+else()
+    include(FindPkgConfig)
+    pkg_search_module(Z3 REQUIRED z3)
+	set(Z3_INSTALLS )
+	set(Z3_EXTRA_LINK_DIR )
+endif()
+
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include
-                    ${CMAKE_CURRENT_BINARY_DIR}/include
-                    ${Z3_DIR}/include/)
```

Comment:
I think we need to include `${Z3_INSTALLS} ` here.

Reply:
This should be fixed.